public/js/theme.js

```diff
@@ -3,7 +3,7 @@
  * Copyright 2018-2020 Medium Rare (undefined)
  */
 ! function(t, e) {
-    "object" == typeof exports && "undefined" != typeof module ? e(exports, require("aos"), require("jquery"), require("jquery-countdown"), require("scrollmonitor"), require("flickity"), require("ion-rangeslider"), require("isotope-layout"), require("jarallax"), require("plyr"), require("prismjs"), require("smooth-scroll"), require("@tanem/svg-injector"), require("twitter-fetcher"), require("typed.js")) : "function" == typeof define && define.amd ? define(["exports", "aos", "jquery", "jquery-countdown", "scrollmonitor", "flickity", "ion-rangeslider", "isotope-layout", "jarallax", "plyr", "prismjs", "smooth-scroll", "@tanem/svg-injector", "twitter-fetcher", "typed.js"], e) : e((t = t || self).theme = {}, t.AOS, t.jQuery, null, t.scrollMonitor, t.flatpickr, t.Flickity, null, t.Isotope, t.jarallax, t.Plyr, t.Prism, t.SmoothScroll, t.SVGInjector, t.Typed)
+    "object" == typeof exports && "undefined" != typeof module ? e(exports, require("aos"), require("jquery"), require("jquery-countdown"), require("scrollmonitor"), require("flickity"), require("ion-rangeslider"), require("isotope-layout"), require("jarallax"), require("plyr"), require("prismjs"), require("smooth-scroll"), require("typed.js")) : "function" == typeof define && define.amd ? define(["exports", "aos", "jquery", "jquery-countdown", "scrollmonitor", "flickity", "ion-rangeslider", "isotope-layout", "jarallax", "plyr", "prismjs", "smooth-scroll", "typed.js"], e) : e((t = t || self).theme = {}, t.AOS, t.jQuery, null, t.scrollMonitor, t.flatpickr, t.Flickity, null, t.Isotope, t.jarallax, t.Plyr, t.Prism, t.SmoothScroll, t.Typed)
```

Comment:
Removing `t.SVGInjector` produces no dropdown triggering in the navigation. Actually I think `t.SVGInjector` should be deleted for real, but the error comes from bdf388b85eab8d0d13d7f62928bcec4bc082a0c1 - the flatpicker.js is not deleted entirely from the project. In commit 9d5b8ba6e9c5f347ba0daabba9c58399b2235813 I made it easy to read that strange thing is an ternary and the arguments number should be the same